File: app/src/main/java/com/home/dab/justdemo/TestFragment.java
package com.home.dab.justdemo;

import android.view.View;

import com.dab.just.base.LazyFragment;

import org.jetbrains.annotations.Nullable;

/**
 * Created by dab on 2018/1/6 0006 12:51
 */

public class TestFragment extends LazyFragment{
    @Override
    protected void onFirstVisibleToUser(@Nullable View view) {

    }

    @Override
    protected int viewLayoutID() {
        return R.layout.fragment_text;
    }
}

File: app/src/main/java/com/home/dab/justdemo/TestFragment.kt
package com.home.dab.justdemo

import android.view.View

import com.dab.just.base.LazyFragment

/**
 * Created by dab on 2018/1/6 0006 12:51
 */

class TestFragment : LazyFragment() {
    override fun onFirstVisibleToUser(view: View?) {

    }

    override fun viewLayoutID(): Int = R.layout.fragment_text
}
